tests/functional/basic/source/data/in/tables/sample1.csv

```diff
@@ -0,0 +1,5 @@
+"someText"
+"this","row","is","invalid"

```

Comment:
není "invalid", ne? je to trochu zavádějící

Reply:
beru to jakoze je invalid, ze ma vic sloupcu nez ten prvni